{{cookiecutter.project_slug}}/Makefile

```diff
@@ -175,11 +181,14 @@ setup-cicd-poetry:
 	. $(HOME)/.poetry/env && poetry config virtualenvs.create false
 	echo "WARNING: you still need to source $$HOME/.poetry/env to access poetry's executable"
 
+{% if cookiecutter.hosting == "GitLab" -%}
 setup-cicd-test-stage-gitlab: setup-cicd-poetry
 
 setup-cicd-release-stage-gitlab: setup-cicd-python3 setup-cicd-poetry setup-release-tools-gitlab
+{%- else %}
 
```

Comment:
`setup-cicd-test-stage-github` not implemented

Reply:
No need... changed the code.